### DISCUSSION TITLE
Nested tabs?

### DISCUSSION THREAD

**Discussion by josecarre** [kinghat/tabbed-card#121] [Q&A]
Is there any way to have nested tabs? Like an upper tab bar like now... but for some of the tabs be able to have a secondary row with more tabs?
Thanks!

**Comment by ildar170975** [comment]
any reasons of not using tabbed-card inside each tab?

**Comment by josecarre** [comment]
Yeah... use case is for thermostat card, i'd like to have inside that card (that form part of a view tab with all other climate stuff) 2 main tabs, one for heat (that shows just the Heat nest thermostat) and other tab for A/C, that when selected shows 5 cards, one for each room where I have a thermostat.

So ONE card with 2 tabs... first tab shows this:
![image](https://user-images.githubusercontent.com/10371413/227249320-b817507d-3dfa-49ba-b2f1-a57e1a8ef28b.png)
And second tab shows something like this:
![image](https://user-images.githubusercontent.com/10371413/227249405-596fc3e5-79e1-4ccd-82b1-569e98ee3865.png)

**Comment by kinghat** [comment]
like @ildar170975 suggested, why not just put a tabbed-card in a tab?
